### ISSUE TITLE
texture bokshelves bug

### ISSUE THREAD

**Issue by FrapzHaash** [Madis0/fabulously-optimized#620]
### Describe the bug

I use a texture pack with some library textures, with optifine it works perfectly

### Modpack version

4.6.0 for 1.19.3 

### Launcher

Minecraft Launcher (vanilla)

### Other things

- [X] I updated FO to a newer version (mention, what version to what version)
- [X] I added/removed some mods or resource packs (mention, which ones)

### Additional context

![image](https://user-images.githubusercontent.com/80805708/229624401-9077067b-e3f9-4b57-8c5c-bdcb798852ec.png)
I removed some mods but I don't remember which ones
![image](https://user-images.githubusercontent.com/80805708/229624648-26f58554-b3b7-4e4f-9d56-79c9898c3210.png)

**Comment by Julienraptor01** [comment]
what's the resource pack ?

**Comment by FrapzHaash** [comment]
create my own resource pack by gathering content from many others, but i cant upload here for its size

**Comment by Julienraptor01** [comment]
> i create my own resource pack by gathering content from many others, link (https://drive.google.com/file/d/1RBcpGjvYe-4VBSet-XYQdAuJluj_yI8i/view?usp=sharing), too big to upload to github

won't look at it RN cause i'm on 4G but have you updated it after 1.19.2 ? 1.19.2 does have breaking changes in how RP are so to be fully supported many packs need an update (just trying clearing an easy problem with an easy solution just in case)

**Comment by FrapzHaash** [comment]
I didn't update it, how should I do it?

**Comment by Julienraptor01** [comment]
[Here's a page that briefly explain what's the change that happened in 1.19.3](https://minecraft.fandom.com/wiki/Java_Edition_1.19.3)
TL;DR some folders sometimes need to be moved in sub-folders etc
i can take a look tmr if that's the case for your pack and maybe fix it if i have the time
the reason it work on OF is probably because OF just restore the <=1.19.2 system of resource loading (which was changed for perf reasons...) or has a custom patch to allow the old packs to work

**Comment by FrapzHaash** [comment]
I would really apreciate it :wink:

**Comment by Julienraptor01** [comment]
So i found out that you used Vanilla Tweaks Connected Variated Bookshelves in your pack and other stuff from the VT Team.
It would take me too long to just try to remake your pack but with all the new stuff so what i recommend you do is just grab the newer versions of all the packs you merged and just recreate your pack but with the updated versions.
